library/src/options.cxx

```diff
@@ -116,7 +116,9 @@ options::options()
   this->Internals->init("render.show-edges", false);
   this->Internals->init("render.line-width", 1.0);
   this->Internals->init("render.point-size", 10.0);
-  this->Internals->init("render.grid", false);
+  this->Internals->init("render.grid.enable", false);
+  this->Internals->init("render.grid.unit", 0.0);
```

Comment:
shouldnt the default be negative ?

Reply:
It gets recomputed if `unitSquare <= 0` so zero does work as a "not set" value (while also being a valid distance value but that's pretty deep nitpicking).

It can be set to `-1` here if it makes it easier to read/maintain, let me know.

Reply:
Right, non-positive.